docs/src/user-guide/core-clp-s.md

```
@@ -74,13 +99,14 @@ Usage:
 Usage:
 
 ```shell
-./clp-s s [<options>] <archives-dir> <kql-query>
+./clp-s s [<options>] <archives-path> <kql-query>
 ```
 
-* `archives-dir` is a directory containing archives.
+* `archives-path` is a directory containing archives, a path to an archive, or a URL pointing to a 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)